src/Lumi/Styles/Box.purs

```diff
@@ -0,0 +1,64 @@
+module Lumi.Styles.Box where
+
+import React.Basic.Emotion (Style, class IsStyleProperty, css, merge, prop, str)
+
+box :: Style
+box =
+  css
+    { display: str "flex"
+    , flexDirection: str "column"
+    , boxSizing: str "border-box"
+    , minHeight: str "0"
+    , flex: str "0 0 auto"
+    }
+
+column :: Style
+column = box
+
+row :: Style
+row =
+  merge
+    [ box
+    , css { flexDirection: str "row" }
+    ]
+
+wrap :: Style
+wrap =
+  css
+    { flexWrap: str "wrap"
+    }
+
+data FlexAlign
+  = Start
+  | End
+  | Center
+  | Stretch
+  | Baseline
+  | Unset
+
+instance isStylePropertyFlexAlign :: IsStyleProperty FlexAlign where
+  prop a =
+    str case a of
+      Start -> "flex-start"
+      End -> "flex-end"
+      Center -> "center"
+      Stretch -> "stretch"
+      Baseline -> "baseline"
+      Unset -> "unset"
```

Comment:
Presumably this is not the final version of this instance? `FlexAlign` is used for `justifyContent`, but that property doesn't accept this set of values (e.g. `baseline` isn't valid, and `space-between`, `space-around`, etc. are missing).

Reply:
Yeah, it's hacky. It was more of an experiment of whether an `IsStyleProperty` class made sense. I'm not sure it really helps unless it's well known and easier to reach for than `str "flex-start"`. I didn't want to spend time trying to type CSS much at this point though.

Reply:
Got it! Also if we ever want to have typed CSS I believe we could use [purescript-css](https://pursuit.purescript.org/packages/purescript-css/3.0.0)

Reply:
Yeah, I looked at it a little bit. It might be a little difficult to work into this API since it focuses on building CSS strings but it's probably doable. I don't know that it seems very nice to use though. I mostly wanted to avoid accidentally passing Maybes or Colors as style values and the StyleProperty type does that.